### ISSUE TITLE
Failed Build: Unable to locate package snapcraft

### ISSUE THREAD

**Issue by jimafisk** [plentico/plenti#347]
Build: https://github.com/plentico/plenti/actions/runs/11524578787/job/32085312420

Error:
```
E: Unable to locate package snapcraft
Error: Process completed with exit code 100.
```

I believe this is related to a fix for the `serve` cmd on Linux releases (https://github.com/plentico/plenti/issues/323) that's potentially related to an older version of `glibc` being used in the build process because it was referencing Ubuntu 20.04. After upgrading to Ubuntu 24.04 (https://github.com/plentico/plenti/commit/a9ec54a5e959b6f61b4cafaa3ca1f2767e7ebb41) the step for setting up snaps fails, potentially because snapcraft is already included:

https://github.com/plentico/plenti/blob/a9ec54a5e959b6f61b4cafaa3ca1f2767e7ebb41/.github/workflows/release.yml#L24

**Comment by jimafisk** [comment]
Was still failing: https://github.com/plentico/plenti/actions/runs/11598485945/job/32294435295

So attempted to remove snapcraft again: https://github.com/plentico/plenti/commit/1a23191aeb9284be14726048ab61c117d06d12da

**Comment by jimafisk** [comment]
After some adjustments for another reason builds were failing (related to llvm): https://github.com/plentico/plenti/issues/349

We're having snapcraft issues again:

```
release failed after 1m28s               error=snapcraft not present in $PATH
Error: The process '/opt/hostedtoolcache/goreleaser-action/2.3.2/x64/goreleaser' failed with exit code 1
```

Failed build: https://github.com/plentico/plenti/actions/runs/11599016707/job/32296214897

**Comment by jimafisk** [comment]
Now I see the key part in the failed build logs: https://github.com/plentico/plenti/actions/runs/11599142938/job/32296623627

```
No apt package "snapcraft", but there is a snap with that name.
Try "snap install snapcraft"
```

**Comment by jimafisk** [comment]
Failed build: https://github.com/plentico/plenti/actions/runs/11599239368/job/32296932634

```
error: access denied (try with sudo)
Error: Process completed with exit code 1.
```

**Comment by jimafisk** [comment]
Should be fixed in [v0.7.14](https://github.com/plentico/plenti/releases/tag/v0.7.14)
